unicodetools/src/main/java/org/unicode/tools/emoji/ProposalData.java

```diff
@@ -317,6 +317,12 @@ public static Set<String> cleanProposalString(String proposalString) {
             ImmutableMap.<String, String>builder()
                     .put("🧑🏿‍❤️‍💋‍🧑🏿", "💏🏿")
                     .put("🧑🏿‍❤️‍🧑🏿", "💑🏿")
+                    .put("👨🏿‍🐰‍👨🏿", "👯🏿")
```

Comment:
I’m still not entirely sure what distinguishes these from handshakes, but it works for now.